styles-conference/assets/stylesheets/styles.css

```diff
@@ -668,3 +683,29 @@ textarea {
     transform: scale(1.2) rotate(360deg);
   }
 }
+
+.blend-mode {
+  filter: blur(8px);
+}
+
+.toggle-menu {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background-color: #6070ff;
+  margin-top: 0;
+  mix-blend-mode: multiply;
+  list-style: none;
+  padding-left: 0;
+  z-index: 1;
+}

```

Comment:
[OPTIONAL] I think it would give your site that killer touch if you were to incorporate a **transition** that occurs when you open and close your menu 😉, transitions make the website feel and look natural to users.